src/v/cluster/cluster_utils.cc

```diff
@@ -109,7 +109,30 @@ ss::future<> maybe_create_tcp_client(
                       std::optional<ss::tls::credentials_builder> credentials) {
                   if (credentials) {
                       return credentials
-                        ->build_reloadable_certificate_credentials();
+                        ->build_reloadable_certificate_credentials(
+                          [](
+                            const std::unordered_set<ss::sstring>& updated,
+                            const std::exception_ptr& eptr) {
+                              if (eptr) {
```

Comment:
it looks like this can also be handled by `log_certificate_reload_event`? perhaps it needs to move to a common location so it can be used here and in application?

Reply:
done

Reply:
moved `log_reloadable_certificate_credentials` to cluster_utils, since application.cc depends on it